addons/web/static/src/js/fields/relational_fields.js

```diff
@@ -1800,6 +1800,7 @@ var FieldMany2ManyBinaryMultiFiles = AbstractField.extend({
                     });
                 }
             }
+            file.upload = true;
```

Comment:
Seems strange, as the old widget did not need that information, why do you need it now ?

Reply:
removed